core/dbt/adapters/base/impl.py

```diff
@@ -1162,6 +1162,47 @@ def get_rows_different_sql(
 
         return sql
 
+    def valid_incremental_strategies(self):
+        return ["append", "delete+insert"]
```

Comment:
I _think_ this should just be `"append"` by default, since that's "true neutral".

In Postgres, we should override it to what's here:
```python
    def valid_incremental_strategies(self):
        return ["append", "delete+insert"]
```
Let's also add a docstring to the effect of: "The set of standard builtin strategies which this adapter supports out-of-the-box. Not used to validate custom strategies defined by end users."

(cc @dataders)

Reply:
Done!

Reply:
Thanks for adding the docstring! Going to be super helpful to maintainers of other adapters.

Could we update this to just `"append"` (default), and then override to `["append", "delete+insert"]` for Postgres?

Reply:
Actually done now...